recipe/meta.yaml

```diff
@@ -17,16 +17,16 @@ build:
   detect_binary_files_with_prefix: true
   run_exports:
     - {{ pin_subpackage("re2", max_pin="x.x.x") }}
+  missing_dso_whitelist:
+    - '$RPATH/ld64.so.1'
 
 requirements:
   build:
     - cmake
-    - make
-    - ninja
+    - make  # [unix]
+    - ninja # [unix]
```

Comment:
shouldn't this be ninja-base ?

Reply:
Thanks ninja-base seems to be sufficient.